components/projects/upload/LinkForm/index.tsx

```diff
@@ -65,6 +65,7 @@ const LinkForm: FC = () => {
               <Controller
                 control={control}
                 name={`links.${index}.url`}
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
                 render={({ field: { value, onBlur, onChange, ...props } }) => (
```

Comment:
예외로 두지 말고, 사용하지 않는 변수를 제거하자!

```suggestion
                render={({ field: { value, onChange, ...props } }) => (
```

Reply:
저 onBlur 지우면 바로 밑의 onBlur에서 오류 나더라구요
<img width="807" alt="image" src="https://user-images.githubusercontent.com/73823388/192824381-f5581a92-a591-4aee-ba5b-95f56fa381aa.png">

제가 react-hook-form을 안 써봐서 다른 해결 방법이 있는지 지금은 잘 모르겠습니다 ㅜ